### ISSUE TITLE
Move OracleDetail in child components

### ISSUE THREAD

**Issue by thegreat-vanderlinde** [vyper-protocol/vyper-otc-ui#386]
Here is not useful having the `OracleDetail` object
https://github.com/vyper-protocol/vyper-otc-ui/blob/7e9b504bfb22d77879d5e323b0d43bbea412800a/src/pages/contract/create/index.tsx#L71-L98

The parent component (contract create page) is interested in knowing only the `oracle pubkey` and the `oracle type`, all the metadata labels/urls is something that should be managed only by child components responsible of showing those information.

**Comment by vantessel** [comment]
we could even store all the contract defaults in one single file and then those are reset whenever the create page is loaded, unless it comes from a template / cloning, wdyt?

**Comment by thegreat-vanderlinde** [comment]
yes, definitely. Having a single source of truth for a new contract default is a good practice. on the contract create page we should check if there some cloning/template data, if not we read the initial state from the default value. could be on a separate file or simply a `const` object at the file beginning. related to #387
